### PR TITLE
Feature - Add weekly summary for users and program

### PR DESCRIPTION
**Pull request by ricconoel**
Changes:
- Add new SQL scripts
- Create functions in query.py to call the new SQL scripts
- Call new functions in main.py

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
 so_ids=[19371698,19430638,19371700,18241184,19371813,16895640,14733669,17720354,19371728,19372058,16895625,19371701,19229284,17720353,19229284,19378437,19378826]
 BUCKET_NAME="so-dev-tracker"
 PROJECT_ID="tiph-ricconoel-batch8"
-DATASET_ID="so_dataset"
+#DATASET_ID="so_dataset"
+DATASET_ID="so_dataset_dev"
```
Comment (ricconoel) [inline constants.py:5]: NOTE: Revert back to `so_dataset` prior to merging

File: main.py [hunk 2]
```diff
@@ -120,6 +120,13 @@ def query_runner(table_name_suffix):
     total_view_id = query.create_total_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
     summary_view_id = query.create_summary_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
 
+    weekly_answer_view_id = query.create_weekly_answer_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
+    weekly_comment_view_id = query.create_weekly_comment_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
+    weekly_total_view_id = query.create_weekly_total_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
+    weekly_summary_view_id = query.create_weekly_summary_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
+    prog_weekly_summary_view_id = query.create_prog_weekly_summary_view(project_id=constants.PROJECT_ID,dataset_id=constants.DATASET_ID,year_month=table_name_suffix)
+
+
```
Comment (ricconoel) [inline main.py:129]: Remove white space

**Review by ricconoel — COMMENTED** [review]
Make sure to check resolve review prior to merging

**Comment by ricconoel** [comment]
Updated weekly_summary_view.sql

**Comment by ricconoel** [comment]
Hi @anjelab04 I updated the scripts. Please review. Thanks!

File: constants.py (hunk 1 above)
Comment (anjelab04) [reply constants.py:5]: looks good to me

**Review by anjelab04 — APPROVED** [review]

**Comment by ricconoel** [comment]
The following are the release steps to be followed:
1. Update `constants.py` to point to `so_dataset`
2. Update commit changes to repository
3. Merge pull request (https://github.com/ricconoel/so_metrics/pull/11)
4. Run `python main.py` to create the new weekly views and update the production data
5. Check if views were created

Will release the changes tomorrow (2022-11-23).

**Comment by ricconoel** [comment]
Will now proceed with the release

**Pull request merged**